src/main/java/uk/ac/ebi/ampt2d/metadata/persistence/repositories/AssemblyRepository.java

```diff
@@ -17,20 +17,23 @@
  */
 package uk.ac.ebi.ampt2d.metadata.persistence.repositories;
 
-import org.springframework.data.repository.CrudRepository;
-import org.springframework.data.repository.query.Param;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.querydsl.QueryDslPredicateExecutor;
+import org.springframework.data.querydsl.binding.QuerydslBinderCustomizer;
+import org.springframework.data.querydsl.binding.QuerydslBindings;
 import org.springframework.data.rest.core.annotation.RepositoryRestResource;
 import uk.ac.ebi.ampt2d.metadata.persistence.entities.Assembly;
-
-import java.util.List;
+import uk.ac.ebi.ampt2d.metadata.persistence.entities.QAssembly;
 
 @RepositoryRestResource
-public interface AssemblyRepository extends CrudRepository<Assembly, Long> {
-
-    List<Assembly> findByName(@Param("name") String name);
-
-    List<Assembly> findByNameAndPatch(@Param("name") String name, @Param("patch") String patch);
+public interface AssemblyRepository extends JpaRepository<Assembly, Long>,
```

Comment:
I think it still works with CrudRepository

Reply:
Yes, it still works with CrudRepository. I have changed it back.